### ISSUE TITLE
Date trait save button will overwrite data with current date even if data is already present

### ISSUE THREAD

**Issue by trife** [PhenoApps/Field-Book#1028]
Related to #519

**Comment by bshallwachs** [comment]
Yes, I have this issue too. Also, when I combine data sheets and upload back into field book, when I am viewing a specific trait i.e. date sown in greenhouse, the new upload does not allow me to see the populated data in the individual accession view. When I switch to the large grid to see all the data, I can see the previous data I uploaded. I am afraid this will overwrite my existing data.

**Comment by trife** [comment]
This will be fixed in v6.1.

Importing past data into Field Book isn't currently supported, so I'm not entirely sure what behavior you're seeing there but feel free to make a separate issue with more information. You can also reach out over email to discuss further.

**Comment by bshallwachs** [comment]
@trife . That answers my question, it isn't supported :). Thank you for your help.
